CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Add `iam:ListRoleTags` and `iam:UntagRole` permissions to the AWS operator role.
+- CAPA: add `ec2:DeleteVolume` for volume cleanup.
```

Comment:
Can you please make this more specific? CAPA doesn't seem to use the `DeleteVolume` request.

If this is solely for mc-bootstrap, for example, let's instead create a new policy file `mc-bootstrap-policy.json`, even if for historical reasons we still attach that to the CAPA controller IAM role for now (or forever).

Reply:
It is mainly for the mc-bootstrap now, but since Capa is planning to clean EBS volumes in their external garbage collector I thought it would be better to have this under the Capa controller IAM
https://cluster-api-aws.sigs.k8s.io/topics/external-resource-gc.html

Reply:
This isn't yet part of CAPA's GC features (see `cluster-api-provider-aws/pkg/cloud/services/gc/`). Let's keep this separated, please. I also have a use case where all S3 buckets must be listed in order to clean them up via mc-bootstrap – CAPA will likely never need that permission and therefore we should least split these permissions into a separate policy. Same for what we need from Crossplane: I shouldn't spill that into the CAPA policy since otherwise we'll never be able to tell which permissions CAPA needs, and which are used for other features. It's much easier if we can diff `capa-controller-role/capa-controller-policy.json` against upstream and not have our own stuff in there.